srcStatic/ConsoleArgumentParser.cpp

```diff
@@ -29,7 +29,7 @@ std::string FindModuleDirectory(std::vector<std::string> arguments)
 		const std::string switchName = GetSwitch(arguments);
 
 		if (switchName.empty()) {
-			return std::string();
+			return "";
```

Comment:
I specifically wrote std::string() to indicate an empty string and prefer it. Although I don't mind if you change it. I have been inconsistent with the application in other parts of the code base. With the quotes, I feel like I don't know for sure you are returning a std::string without looking at the function return type.

https://stackoverflow.com/questions/26587110/returning-an-empty-string-efficient-way-in-c/26588207

Reply:
Hmm, very interesting link. It seems `std::string()` and `{}` compile to the same, while `""` can compile to less efficient code under certain compiler optimization settings. I checked it out for a few compilers and settings on: https://godbolt.org/

I think I'll switch it back to `std::string()`. That's certainly more readable than `{}`.